homework/week-0/AllisonBojarski/AnimalSort(HWproblem#1)/src/nyc/c4q/alizinha/Animal.java

```diff
@@ -0,0 +1,30 @@
+package nyc.c4q.alizinha;
+/**
+ * Created by c4q-Allison on 5/1/15.
+ */
+public class Animal implements Comparable<Animal>
+{
+    String species;
+
+    public String getSpecies()
+    {
+        return species;
+    }
+
+    @Override
+    public int compareTo(Animal o)
+    {
+        return this.species.compareTo(o.getSpecies());
```

Comment:
As 'Comparable' is an interface, this @Override method should compute the comparison based on the assignment specs and/or the [documentation](<https://docs.oracle.com/javase/7/docs/api/java/lang/Comparable.html>) specs.